lib/compat/wordpress-6.4/theme-previews.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Enable theme previews in the Site Editor for block themes.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Allow developers to customize the back to dashboard link in the Site Editor
+ *
+ * @param array $settings The editor settings.
+ * @return array The editor settings.
+ */
+
+function gutenberg_theme_preview_block_editor_settings_all( $settings ) {
+	$settings['__experimentalDashboardLink'] = 'themes.php';
+	return $settings;
+}
+
+// Attaches filters to enable theme previews in the Site Editor.
```

Comment:
📝 This would go inside of `initialize_theme_preview_hooks` to avoid the [global scope](https://core.trac.wordpress.org/ticket/59000#comment:15) when we port this to the core.
https://github.com/WordPress/wordpress-develop/blob/e14213e/src/wp-includes/theme-previews.php#L86